server/service.go

```diff
@@ -337,16 +342,15 @@ func (m *BoostService) handleGetHeader(w http.ResponseWriter, req *http.Request)
 		"slotTimeSec": config.SlotTimeSec,
 		"msIntoSlot":  msIntoSlot,
 	}).Infof("getHeader request start - %d milliseconds into slot %d", msIntoSlot, _slot)
-
 	// Add request headers
 	headers := map[string]string{
-		HeaderKeySlotUID: slotUID.String(),
+		HeaderKeySlotUID:          slotUID.String(),
+		HeaderStartTimeMsIntoSlot: fmt.Sprintf("%d", msIntoSlot),
```

Comment:
why do we need to add msIntoSlot in the header? `HeaderStartTimeMsUnix` should contain the same information

Reply:
Ok. I have removed `HeaderStartTimeMsUnix` from getHeader adn getPayload.

Reply:
I think you should remove `HeaderStartTimeMsIntoSlot` and have `HeaderStartTimeMsUnix` consistently across the endpoints